pkg/consoleuser/consoleuser_test.go

```diff
@@ -0,0 +1,38 @@
+package consoleuser
+
+import (
+	"context"
+	"os"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCurrentUids(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
```

Comment:
Same question as above?

Maybe modulated a little about if it's interesting to encode expectations about the platforms into the test table, or create platform specific test files or ???

Reply:
I dropped the table tests ... despite my earlier comment about "consistency", it does look better and is easy to parse

Reply:
I do the same thing sometimes. I see a test without tables, and I'm like "whoa! Hard to read" then I read it and remember it's what would have been in the `t.Run`.